### PR TITLE
feat: About CoMapeo screen

### PR DESCRIPTION
**Pull request by ErikSin**
Created About CoMapeo Screen

Also created a wrapper hook, that uses react query to wrap [react-native-device-info](https://www.npmjs.com/package/react-native-device-info)

image

### CODE REVIEW

**Comment by ErikSin** [comment]
related to: https://www.notion.so/digidem/Can-t-find-About-0001387e323b4980b923bc4fc31c8bbc

**Review by cimigree — CHANGES_REQUESTED** [review]
Logic looks great! So tidy. Nice wrapper! My questions are all related to Mapeo naming. Also, if I update the language, the language on the screen doesn't change -- maybe because of the difference in name.

File: messages/en.json [hunk 1]
```diff
@@ -156,6 +156,38 @@
...
+  "screens.AboutMapeo.androidBuild": {
+    "description": "Label for Android build number",
+    "message": "Android build number"
+  },
+  "screens.AboutMapeo.androidVersion": {
+    "description": "Label for Android version",
+    "message": "Android version"
+  },
+  "screens.AboutMapeo.mapeoBuild": {
+    "description": "Label for Mapeo build number",
```
[the remark this answers is not in the document]
Comment (ErikSin) [reply messages/en.json]: https://github.com/digidem/comapeo-mobile/pull/432/commits/67bdb331f14580ac35c7572f2edd35fefa7316ea

File: src/frontend/screens/Settings/About.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+import {ScrollView} from 'react-native-gesture-handler';
+import {defineMessages, useIntl} from 'react-intl';
+
+import {List, ListItem, ListItemText} from '../../sharedComponents/List';
+import {MethodName, useDeviceInfo} from '../../hooks/useDeviceInfo';
+import {UIActivityIndicator} from 'react-native-indicators';
+
+const m = defineMessages({
+  aboutMapeoTitle: {
+    id: 'screens.AboutMapeo.title',
```
[the remark this answers is not in the document]
Comment (ErikSin) [reply src/frontend/screens/Settings/About.tsx:11]: https://github.com/digidem/comapeo-mobile/pull/432/commits/67bdb331f14580ac35c7572f2edd35fefa7316ea

**Review by cimigree — APPROVED** [review]
Language is still not changing, but I think that is unrelated to this PR

**Comment by ErikSin** [comment]
> Language is still not changing, but I think that is unrelated to this PR

yeah, not related. The translation has not need added to crowdin yet (because it has just been introduced in this pr), so there are not translations available.

**Pull request merged**